daemon/mgr/network_utils.go

```diff
@@ -2,49 +2,12 @@ package mgr
 
 import (
 	"fmt"
-	"strings"
 
 	"github.com/alibaba/pouch/apis/types"
 
 	"github.com/docker/libnetwork"
 )
 
-// IsContainer is used to check if network mode is container mode.
-func IsContainer(mode string) bool {
-	parts := strings.SplitN(mode, ":", 2)
-	return len(parts) > 1 && parts[0] == "container"
-}
-
-// IsHost is used to check if network mode is host mode.
-func IsHost(mode string) bool {
```

Comment:
Like these function is repeat with pid or user or .. namespace, this is not suitable in a network_util.go